src/frontend/Sider/Behandling/VedtakOgBeregning/Barnetilsyn/InnvilgeVedtak/Vedtaksperioder.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { useState } from 'react';
+
+import styled from 'styled-components';
+
+import { PlusCircleIcon } from '@navikt/aksel-icons';
+import { Button, Heading, Label, VStack } from '@navikt/ds-react';
+
+import { useApp } from '../../../../../context/AppContext';
+import { useSteg } from '../../../../../context/StegContext';
+import { FormErrors } from '../../../../../hooks/felles/useFormState';
+import { UlagretKomponent } from '../../../../../hooks/useUlagredeKomponenter';
+import { VedtaksperiodeTilsynBarn } from '../../../../../typer/vedtak/vedtakTilsynBarn';
+import { tomVedtaksperiode } from '../VedtakBarnetilsynUtils';
+import { VedtaksperiodeRad } from './VedtaksperiodeRad';
+
+const Grid = styled.div`
+    display: grid;
+    grid-template-columns: repeat(5, max-content);
+    grid-gap: 0.5rem 1.5rem;
+    align-items: start;
+
+    > :nth-child(5n) {
+        grid-column: 1;
+    }
+`;
+
+interface Props {
+    vedtaksperioder: VedtaksperiodeTilsynBarn[];
+    settVedtaksperioder: React.Dispatch<React.SetStateAction<VedtaksperiodeTilsynBarn[]>>;
```

Comment:
For at vi skal validere vedtaksperioder riktig så burde vi nog håndtere vedtaksperioder på lik måte som her https://github.com/navikt/tilleggsstonader-sak-frontend/pull/697
Men den endringen kan gjøres etterpå